package.json

```diff
@@ -45,8 +45,8 @@
   "author": "Aaron Franks",
   "license": "MIT",
   "dependencies": {
-    "chalk": "^2.4.1",
-    "dotenv": "^6.2.0",
+    "chalk": "^2.4.2",
+    "dotenv": "^8.2.0",
```

Comment:
this has dropped node v6, so when we merge this it should be released in a major

Reply:
Node 8.12 is already required since https://github.com/SimenB/envalid/pull/12